.github/workflows/docker-build.yml

```
@@ -25,6 +25,10 @@ jobs:
             type=edge,branch=main
           flavor: |
             latest=auto
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@master
```

To prevent future surprises...

```suggestion
        uses: docker/setup-qemu-action@v1
```